app/src/main/res/layout/main_fragment.xml

```diff
@@ -37,21 +35,54 @@
 
     <TextView
         android:id="@+id/state_text"
-        android:textSize="18sp"
-        android:text="Hello"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:text="Hello"
+        android:textSize="18sp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/creat" />
 
-    <androidx.recyclerview.widget.RecyclerView
-        android:id="@+id/persons"
+    <androidx.core.widget.NestedScrollView
```

Comment:
I don't get real need in putting ListView (RecyclerView) inside a ScrollView. It's look like a dirty hack to fix some combination of bugs.